src/myr/reference.js

```diff
@@ -135,6 +135,28 @@ let transformations = [
         description: <span>Changes the position of the cursor along the Z axis to the given value.</span>,
         example: "setZPos"
     },
+    {
+        name: <span> increasePosition(<span className="number">x</span>,<span className="number">y</span>,<span className="number">z</span>)</span>,
+        description: <span>Changes the position of the cursor relative to the current position. This function takes both positive and negative values. </span>,
+        example: "increasePosition"
```

Comment:
Can you remove the example line? I'll add that myself once the examples are in the database.